### ISSUE TITLE
Can you please provide some training details of C3D model?

### ISSUE THREAD

**Issue by scream7** [albertomontesg/keras-model-zoo#11]
Thanks for your work firstly,

I'm now training my own C3D model used UCF-101 datasets, and I also fine tune your pre-trained model. But I can't get a satisfied result.

it would be appreciated if you could provide some training details? optimizers /loss function, learning rate,  data preprocessing,etc.

Thanks a lot.
